### PR TITLE
typo percon -> percona

### PR DESCRIPTION
**Pull request by michaelcoburn**

### CODE REVIEW

**Review by gen1us2k — DISMISSED** [review]

**Comment by denisok** [comment]
@gen1us2k could you please cherry-pick this one into your PR, so we would go faster ?

**Comment by denisok** [comment]
@michaelcoburn could you please bump chart version to 1.2 ?
https://github.com/percona/percona-helm-charts/blob/main/charts/pmm/Chart.yaml#L5

**Comment by michaelcoburn** [comment]
@denisok done :+1:

**Comment by denisok** [comment]
strange linter is not happy, probably 1.2.0 and if not happy again "1.2.0"

File: charts/pmm/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: pmm
 description: A Helm chart for Percona Monitoring and Management (PMM)
 type: application
-version: 1.0.1
+version: 1.2
```
Comment (denisok) [inline charts/pmm/Chart.yaml:5]:
```suggestion
version: 1.2.0
```

**Review by denisok — CHANGES_REQUESTED** [review]

**Review by denisok — APPROVED** [review]

**Review by tplavcic — APPROVED** [review]

**Pull request merged**